courses/factories.py

```diff
@@ -18,10 +18,18 @@ class CourseFactory(DjangoModelFactory):
     """Factory for Courses"""
     title = fuzzy.FuzzyText(prefix="Course ")
     program = factory.SubFactory(ProgramFactory)
+    position_in_program = factory.Sequence(lambda n: n)
 
     class Meta:  # pylint: disable=missing-docstring
         model = Course
 
+    @classmethod
+    def _setup_next_sequence(cls):
+        try:
+            return Course.objects.latest('position_in_program').position_in_program + 1
+        except Course.DoesNotExist:
+            return 0
+
 
 class CourseRunFactory(DjangoModelFactory):
```

Comment:
It looks like this comment is unaddressed, but I'll just file a ticket for a later PR